nvme.c

```diff
@@ -8295,7 +8293,7 @@ static int submit_io(int opcode, char *command, const char *desc, int argc, char
 		printf("pif             : %02x\n", pif);
```

Comment:
Wouldn't it make sense to move this part to `logging.c` eventually? I would love to see that we extend the logging there to print more of the commands. But this is just an idea for a new side project :)

Reply:
Right so okay I will do this by a separate patch. Thank you.